bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/java/org/openhab/binding/modbus/helioseasycontrols/internal/HeliosEasyControlsActions.java

```diff
@@ -47,60 +47,80 @@ public void setThingHandler(@Nullable ThingHandler handler) {
 
     private void triggerSwitch(String variableName) {
         try {
-            if (handler != null) {
-                handler.writeValue(variableName, "1");
+            if (this.handler != null) {
+                this.handler.writeValue(variableName, "1");
```

Comment:
Revert

Reply:
These changes were the result of  #8755 and should therefore remain.

Reply:
agreed. I meant that this part of the code should not change at all